### ISSUE TITLE
Add postgresql driver to nifi containers

### ISSUE THREAD

**Issue by banzo** [cetic/helm-fadi#3]
We currently use this hack:

```bash
wget https://jdbc.postgresql.org/download/postgresql-42.2.6.jar
kubectl cp ./postgresql-42.2.6.jar fadi/fadi-nifi-0:/opt/nifi/postgresql-42.2.6.jar
rm postgresql-42.2.6.jar
```

It should ideally be an option of the chart where the version can be specified (or the whole jar download path?).

**Comment by banzo** [comment]
See https://github.com/cetic/helm-nifi/issues/5 for progress

**Comment by alexnuttinck** [comment]
Can be solved now by adding:

```yaml
nifi:
  postStart: /opt/nifi/psql; wget -P /opt/nifi/psql https://jdbc.postgresql.org/download/postgresql-42.2.6.jar
```

**Comment by alexnuttinck** [comment]
merged in 0.1.9!
